src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int maxLength = 100;
+    public static final int figuresTypesNumber = 5;
+    private static final int defaultRadius = 10;
+    private static final String defaultColor = Color.WHITE.name();
+    private static final Figure defaultFigure = new Circle(defaultColor, defaultRadius);
```

Comment:
You don`t need this field, you can create default figure in method getDefaultFigure() and invoke this method only

Reply:
but each time i will be creating new object in heap, is this efficient ?

Reply:
i thought objects that are same for each call must be const

Reply:
public Figure getDefaultFigure() {
        return new Circle(defaultColor, defaultRadius);
    } 
    if you are meaning this

Reply:
surely, somehow thought that object must be constant. But in fact it may be changed if the class user wihes